src/accordionpane/README.md

```diff
@@ -0,0 +1,53 @@
+# @dojo/widgets/accordionpane/AccordionPane widget
```

Comment:
have we used the full path in naming these elsewhere? I think just `# AccordionPane` will suffice

Reply:
Yes, we used full paths elsewhere. I'm going to stick with it here to be consistent. If we want to change that, we should change it throughout all components.